.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+  "name": "C# (.NET)",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/powershell:1": {},
+    "ghcr.io/dapr/cli/dapr-cli": {
+      "version": "1.12.0"
+    }
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.vscodeintellicode-csharp",
```

Comment:
recommend changing this to `ms-dotnettools.csdevit`

Reply:
are there reasons to it? had the same recommendation from @baronfel.
Isn't the intellicode extension just the more advanced one?

Reply:
https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp
![image](https://github.com/dotnet/aspire/assets/7604648/0f6370b8-9dbc-4c80-9615-d02303e36da9)
seemed we should install `ms-dotnettools.csdevkit` to keep it working

Reply:
If you open the extension page in visual studio you see it will install the C# extension by itself:

![Image showing the dependencies of the IntelliCode extension including C# extension](https://github.com/dotnet/aspire/assets/11291885/a73ad31e-4d4c-45e2-a8a0-8ad8bac13093)

Reply:
@WeihanLi It's the same like for the C# extension. That extension will install [`.NET Install Tool`](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime) Extension.